strongbox-user-management/src/main/resources/etc/conf/security-users.xml

```diff
@@ -27,7 +27,66 @@
             <encryption-algorithm>plain</encryption-algorithm>
```

Comment:
@AlexOreshkevich : Would you mind renaming this class to `strongbox-security-users.xml`?